site/src/content/docs/components/alerts.mdx

```diff
@@ -140,7 +140,7 @@ As part of Bootstrap’s evolving CSS variables approach, alerts now use local C
 
 <DeprecatedIn version="5.3.0" />
```

Comment:
Since it's going to be a major version, we'll need to get rid (and probably document in the migration guide) all the things that were deprecated in v5.